### PR TITLE
Add policy override for kms_key

### PR DESCRIPTION
**Pull request by thumphries**
Make kms_key.policy accept an IAM document.

### CODE REVIEW

**Comment by thumphries** [comment]
This appears to have pulled in the new release of `terraform-provider-aws` (generator scripts are not version-locked)

File: provider/terrafomo-aws/gen/Terrafomo/AWS/Resource04.hs [hunk 1]
```diff
@@ -6593,7 +6500,7 @@ data LaunchTemplateResource s = LaunchTemplateResource'
     , _disableApiTermination :: TF.Attr s P.Bool
     -- ^ @disable_api_termination@ - (Optional)
     --
-    , _ebsOptimized :: TF.Attr s P.Bool
+    , _ebsOptimized :: TF.Attr s P.Text
```
Comment (nhibberd) [inline provider/terrafomo-aws/gen/Terrafomo/AWS/Resource04.hs:6503]: interesting

**Review by nhibberd — APPROVED** [review]

File: provider/terrafomo-aws/gen/Terrafomo/AWS/Resource04.hs (hunk 1 above)
Comment (thumphries) [reply provider/terrafomo-aws/gen/Terrafomo/AWS/Resource04.hs:6503]: Wonder what caused this? I don't think there were any overrides for it

**Comment by thumphries** [comment]
Have built this successfully

**Pull request merged**